cmd/habitatctl/commands/start.go

```diff
@@ -22,15 +22,44 @@ func parseFlags(args []string) ([]string, []string) {
 	return flags, nonflags
 }
 
+var procName string
```

Comment:
I'm a little confused by this. Is procName supposed to refer to the app IPFS is supporting? (For example, the notes app)